e2e-polybft/framework/test-cluster.go

```diff
@@ -496,16 +503,17 @@ func NewTestCluster(t *testing.T, validatorsCount int, opts ...ClusterOption) *T
 	var err error
 
 	config := &TestClusterConfig{
-		t:               t,
-		WithLogs:        isTrueEnv(envLogsEnabled),
-		WithStdout:      isTrueEnv(envStdoutEnabled),
-		Binary:          resolveBinary(),
-		EpochSize:       10,
-		EpochReward:     1,
-		BlockGasLimit:   1e7, // 10M
-		StakeAmounts:    []*big.Int{},
-		NumberOfBridges: 0,
-		VotingDelay:     10,
+		t:                    t,
+		WithLogs:             isTrueEnv(envLogsEnabled),
+		WithStdout:           isTrueEnv(envStdoutEnabled),
+		Binary:               resolveBinary(),
+		EpochSize:            10,
+		EpochReward:          1,
+		BlockGasLimit:        1e7, // 10M
+		StakeAmounts:         []*big.Int{},
+		NumberOfBridges:      0,
+		VotingDelay:          10,
+		BridgeBatchThreshold: 25,
```

Comment:
This looks way too small for a default test value.

```suggestion
		BridgeBatchThreshold: 100,
```

Reply:
Yes, but in that case, we would need to wait a long time to start executing the rollback batch. For me, this seems like a reasonable value.

Reply:
But you have the `WithThreshold` config function for rollback tests.

Reply:
Yes, you are right.